nucypher/cli/commands/contacts.py

```diff
@@ -0,0 +1,116 @@
+"""
+ This file is part of nucypher.
+
+ nucypher is free software: you can redistribute it and/or modify
+ it under the terms of the GNU Affero General Public License as published by
+ the Free Software Foundation, either version 3 of the License, or
+ (at your option) any later version.
+
+ nucypher is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU Affero General Public License for more details.
+
+ You should have received a copy of the GNU Affero General Public License
+ along with nucypher.  If not, see <https://www.gnu.org/licenses/>.
+"""
+
+import shutil
+
+import click
+import os
+
+from nucypher.characters.control.emitters import StdoutEmitter
+from nucypher.cli.actions.select import select_card
+from nucypher.cli.options import option_force
+from nucypher.cli.painting.policies import paint_single_card, paint_cards
+from nucypher.cli.types import EXISTING_READABLE_FILE
+from nucypher.policy.identity import Card
+
+
+@click.group()
+def contacts():
+    """"Manage character cards"""
+
+
+@contacts.command()
+@click.option('--id', 'card_identifier', type=click.STRING, required=False)
+@click.option('--name', 'card_nickname', type=click.STRING, required=False)
+@click.option('--qrcode', is_flag=True, default=None)
```

Comment:
Heh - don't feel the need to keep the old `qrcode` code - it was just @cygnusv and I trying something

Reply:
I actually think its very cool :sunglasses: